web/src/elements/PageHeader.ts

```diff
@@ -63,69 +84,212 @@ export class PageHeader extends WithBrandConfig(AKElement) {
-                    min-height: 114px;
-                    max-height: 114px;
-                    background-color: var(--pf-c-page--BackgroundColor);
+                    min-height: 6rem;
+                    padding-block: var(--pf-global--spacer--xs);
+                    row-gap: var(--pf-global--spacer--xs);
+
+                    display: grid;
+                    column-gap: var(--pf-global--spacer--xs);
+                    grid-template-columns: [brand] auto [toggle] auto [primary] 1fr [secondary] auto;
+                    grid-template-rows: auto auto;
+                    grid-template-areas:
+                        "brand toggle primary secondary"
+                        "brand toggle description secondary";
+
+                    @media (max-width: 768px) {
+                        row-gap: var(--pf-global--spacer--xs);
+
+                        align-items: center;
+                        grid-template-areas:
+                            "toggle primary secondary"
+                            "toggle description description";
+                        justify-content: space-between;
+                        width: 100%;
+                    }
                 }
-                .pf-c-page__main-section.pf-m-light {
-                    background-color: transparent;
+
+                .items {
+                    display: block;
+
+                    &.primary {
+                        grid-area: primary;
+                        align-content: center;
+                        padding-inline: var(--pf-global--spacer--sm);
+                        padding-block-start: var(--pf-global--spacer--md);
+
+                        .accent-icon {
+                            height: 1em;
+                            width: 1em;
+
+                            @media (max-width: 767px) {
+                                display: none;
+                            }
+                        }
+                    }
+
+                    &.page-description {
+                        grid-area: description;
+                        padding-inline: var(--pf-global--spacer--sm);
+                        padding-block-end: var(--pf-global--spacer--md);
+
+                        @media (min-width: 769px) {
+                            text-wrap: balance;
+                        }
+                    }
+
+                    &.secondary {
+                        grid-area: secondary;
+                        padding-block: var(--pf-global--spacer--lg);
+                        flex: 0 0 auto;
+                        justify-self: end;
+                        padding-inline-end: var(--pf-global--spacer--xl);
+                    }
                 }
-                .pf-c-page__main-section {
-                    flex-grow: 1;
-                    flex-shrink: 1;
+
+                .brand {
+                    grid-area: brand;
+                    background-color: var(--pf-c-page__sidebar--BackgroundColor);
```

Comment:
```suggestion
                    background-color: var(--pf-c-page__sidebar--m-light--BackgroundColor);
```

Reply:
even with that the logo has a gray border it shouldn't have 
![image](https://github.com/user-attachments/assets/ffe44004-4c26-46f0-8145-840a9c7c30b5)